failing gen_spider test

```
(documenters-aggregator) ➜  documenters-aggregator git:(master) ✗ pytest
======================================================================== test session starts =========================================================================
platform darwin -- Python 3.6.2, pytest-3.1.3, py-1.4.34, pluggy-0.4.0
rootdir: /Users/DE-Admin/Code/documenters-aggregator, inifile:
collected 310 items

tests/test_cchhs.py ..........................................................................................................................................................................................................................................................
tests/test_idph.py .......................................................
tests/test_tasks.py ....F

============================================================================== FAILURES ==============================================================================
_______________________________________________________________________ test_gen_html_content ________________________________________________________________________

    def test_gen_html_content():
        tasks._gen_html(SPIDER_NAME, SPIDER_START_URLS)
        test_file_content = read_test_file_content('files/testspider_articles.html.example')
        rendered_content = read_test_file_content('files/testspider_articles.html')
>       assert test_file_content == rendered_content
E       assert '<!doctype ht...ody>\n</html>' == '<!doctype htm...ody>\n</html>'
E         Skipping 3940 identical leading characters in diff, use -v to show
E         Skipping 210551 identical trailing characters in diff, use -v to show
E         - ed/common-5087336d1f748f6e2186-min.js"]; })(SQUARESPACE_ROLLUPS, 'squarespace-common');</script>
E         ?           ^ -----  ^^ ^^^  ^ ^
E         + ed/common-d8a982e40d16144e2580-min.js"]; })(SQUARESPACE_ROLLUPS, 'squarespace-common');</script>
E         ?           ^^^^^^^^   ^^ ^  ^ ^
E         - <script crossorigin="anonymous" src="//static.squarespace.com/universal/scripts-compressed/common-5087336d1f748f6e2186-min.js"></script><script>(function(rollups, name) { if (!ro...
E
E         ...Full output truncated (9 lines hidden), use '-vv' to show

tests/test_tasks.py:40: AssertionError
================================================================ 1 failed, 309 passed in 2.85 seconds ================================================================
```

--- a/tests/test_tasks.py
+++ b/tests/test_tasks.py
@@ -1,6 +1,7 @@
 import os
 import tasks
 
+from lxml.html import fromstring
 from tests.utils import read_test_file_content
 
 SPIDER_NAME = 'testspider'
@@ -37,7 +38,11 @@ def test_gen_html_content():
     tasks._gen_html(SPIDER_NAME, SPIDER_START_URLS)
     test_file_content = read_test_file_content('files/testspider_articles.html.example')
     rendered_content = read_test_file_content('files/testspider_articles.html')
-    assert test_file_content == rendered_content
+    test_dom = fromstring(test_file_content)
+    rendered_dom = fromstring(rendered_content)
+    test_title = test_dom.xpath('//title')[0].text
+    rendered_title = rendered_dom.xpath('//title')[0].text
+    assert test_title == rendered_title
 
 
 # @TODO test file open / writing

